drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -8,6 +8,10 @@
 
 module Types
   class Application::User < Types::BaseObject
+    EXCLUDED_KEYS = ['owner_type', 'enrollment_address_type', 'wip'].freeze
```

Comment:
Is there a better place for this?

Reply:
Yes probably...these are enrollment-related, so I would think more likely `Types::HmisSchema::Enrollment::EXCLUDED_KEYS_FOR_AUDIT`. The User type here would reference that constant to exclude them (And to exclude the `Types::HmisSchema::Client::EXCLUDED_KEYS_FOR_AUDIT` keys as well, and so on). It feels messy but I think its ok for now.